src/coprocessor/dag/expr/scalar_function.rs

```diff
@@ -352,7 +364,6 @@ impl ScalarFunc {
             | ScalarFuncSig::AesDecrypt
             | ScalarFuncSig::AesEncrypt
             | ScalarFuncSig::Char
-            | ScalarFuncSig::Compress
             | ScalarFuncSig::ConcatWS
```

Comment:
```suggestion

```

Reply:
But if i delete this line, it has a conflict.

Reply:
Look at that, it says :This branch has conflicts that must be resolved.

Reply:
What?
What's the problem if you merge it locally manually?

BTW it'll really help if

> rebase (or squash it manually) your PR to make reviewing easier

Reply:
git rebase??

Reply:
This is the fault
```
<<<<<<< master
=======
            | ScalarFuncSig::ConcatWS
>>>>>>> master
```

Reply:
it says :This branch has conflicts that must be resolved.

Reply:
Check your QQ. I've sent a request to add you

Reply:
If you do a `git rebase`, this message will gone.